3_Implementation/documentation/html/search/files_3.js

```diff
@@ -0,0 +1,4 @@
+var searchData=
+[
+  ['readme_2emd_822',['Readme.md',['../_readme_8md.html',1,'']]]
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["readme_2emd_822",['Readme.md',['../_readme_8md.html',1,'']]]
```